src/formBuilder/CardGeneralParameterInputs.js

```diff
@@ -243,6 +248,48 @@ export default function CardGeneralParameterInputs({
           />
         </div>
       </div>
+
+      <div
+        className={classnames('card-entry', {
+          'wide-card-entry': !showObjectNameInput,
+        })}
+      >
+        <h5>
```

Comment:
I’m not sure if Column size is something that should be a top level thing that we show all the time on a permanent basis. Ideally it would go in the Modal I think

Reply:
That seems more fitting, I've moved it to the modal.